modules/dnn/src/layers/proposal_layer.cpp

```diff
@@ -148,11 +148,86 @@ class ProposalLayerImpl : public ProposalLayer
         deltasPermute->finalize(layerInputs, layerOutputs);
     }
 
+#ifdef HAVE_OPENCL
+    bool forward_ocl(InputArrayOfArrays inps, OutputArrayOfArrays outs, OutputArrayOfArrays intnals)
+    {
+        std::vector<UMat> inputs;
+        std::vector<UMat> outputs;
+        std::vector<UMat> internals;
+
+        inps.getUMatVector(inputs);
+        outs.getUMatVector(outputs);
+        intnals.getUMatVector(internals);
```

Comment:
I think it is better to use "intnals" => "internals_"

Reply:
yeah, variable name is changed.